libs/ha/ha_common_libs_k8s.py

```diff
@@ -1197,3 +1198,114 @@ def get_config_value(pod_obj, pod_list=None):
         data = yaml.safe_load(conf_fd)
 
         return True, data
+
+    @staticmethod
+    def setup_mock_monitor(node_obj):
+        """
+        :param node_obj: Object for node
+        :return: Bool
+        """
+        # TODO: Revisit once RPM for mock_health_event_publisher is available
+        LOGGER.info("Get HA pod name for setting up mock monitor")
+        ha_pod = node_obj.get_all_pods(pod_prefix=common_const.HA_POD_NAME_PREFIX)[0]
+        LOGGER.info("Install git inside ha pod %s", ha_pod)
+        try:
+            node_obj.send_k8s_cmd(operation="exec", pod=ha_pod, namespace=common_const.NAMESPACE,
+                                  command_suffix="yum install git -y", decode=True)
+        except IOError as error:
+            LOGGER.error("Failed to install git inside ha pod %s due to error: %s", ha_pod, error)
+            return False
```

Comment:
Can we send error in return statement for assert ?

Reply:
No need to return error as we are already printing it here. And we are not going to take any action on that error, we just want to exit from test if False is returned.

Reply:
> No need to return error as we are already printing it here. And we are not going to take any action on that error, we just want to exit from test if False is returned.

In that case we need to write our own assert statement, thats why if we send error we can give the same in assert statement to keep it common with actual failure reason.